docs/data/base/components/input/UnstyledInputIntroduction/tailwind/index.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { Input } from '@mui/base/Input';
+import { useTheme } from '@mui/system';
+
+function useIsDarkMode() {
+  const theme = useTheme();
+  return theme.palette.mode === 'dark';
+}
+
+export default function UnstyledInputBasic() {
```

Comment:
The name should match the folder.

```suggestion
export default function UnstyledInputIntroduction() {
```

Reply:
Oh, I missed it. I just fixed it thanks!